src/main/kotlin/org/koitharu/kotatsu/parsers/site/madara/ar/LinkManga.kt

```diff
@@ -5,6 +5,6 @@ import org.koitharu.kotatsu.parsers.MangaSourceParser
 import org.koitharu.kotatsu.parsers.model.MangaSource
 import org.koitharu.kotatsu.parsers.site.madara.MadaraParser
 
-@MangaSourceParser("MANGALINKNET", "MangaLink.net", "ar")
-internal class MangalinkNet(context: MangaLoaderContext) :
-	MadaraParser(context, MangaSource.MANGALINKNET, "manga-link.net", pageSize = 10)
+@MangaSourceParser("LINKMANGA", "Link-Manga.com", "ar")
```

Comment:
The source name should not be changed. It is not shown to the user so it is ok to keep the old name if it is the same source on another domain
```suggestion
@MangaSourceParser("MANGALINKNET", "Link-Manga.com", "ar")
```

Reply:
Ok Thanks .